### PR TITLE
Added board detection for UDOO Bolt using check_board_asset_value()

### PR DESCRIPTION
**Pull request by geekguy-wy**
Added board detection for UDOO Bolt using check_board_asset_value() and added it to detect.py

### CODE REVIEW

File: adafruit_platformdetect/board.py [hunk 1]
```diff
@@ -50,6 +50,7 @@ class Board:
 
     def __init__(self, detector):
         self.detector = detector
+        self.board_asset_tag = self.detector.check_board_asset_tag_value().strip()
```
Comment (makermelissa) [inline adafruit_platformdetect/board.py:53]: Let's add the strip() into the check_board_asset_tag_value() function.

File: adafruit_platformdetect/board.py [hunk 2]
```diff
@@ -117,7 +118,7 @@ def id(self):
             board_id = self._clockwork_pi_id()
         elif chip_id == chips.RK3308:
             board_id = self._rock_pi_id()
-        elif chip_id == chips.RYZEN_V1605B:
+        elif self.board_asset_tag == "SC40-2000-0000-C0|C":
```
Comment (makermelissa) [inline adafruit_platformdetect/board.py:121]: Can we move the actual tag into the constants/board.py file? You could make an IDs tuple like we did for other boards. That way if UDOO adds additional IDs, it's easy to add new ones in and it looks cleaner.

**Comment by makermelissa** [comment]
One more thing is adding an `any_udoo_board()` like we did with the others. It'll make adding Blinka support easier.

**Comment by geekguy-wy** [comment]
@makermelissa Do you mean to move both the UDOO_BOLT and board_asset_tag_value into a tuple?

File: adafruit_platformdetect/board.py (hunk 1 above)
Comment (geekguy-wy) [reply adafruit_platformdetect/board.py:53]: Done.

**Comment by makermelissa** [comment]
No, I mean just the hard-coded number

File: adafruit_platformdetect/board.py (hunk 2 above)
Comment (geekguy-wy) [reply adafruit_platformdetect/board.py:121]: Like this?`
'_UDOO_BOLT_ASSET = "SC40-2000-0000-C0|C"

_UDOO_IDS = (
   UDOO_BOLT,
   _UDOO_BOLT_ASSET,
)'

Comment (makermelissa) [reply adafruit_platformdetect/board.py:121]: Not quite. More like:
```
_UDOO_BOARD_IDS = (
    UDOO_BOLT: (
        "SC40-2000-0000-C0|C",
    )
)'
```

**Comment by makermelissa** [comment]
Hi, I submitted a PR to your repo at https://github.com/geekguy-wy/Adafruit_Python_PlatformDetect/pull/1 that should make it so that it doesn't break the code for other boards. Please give that a try and then we can get this merged in soon.

**Pull request merged**